packages/components/form/src/phone_field/phone_field.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  isPossiblePhoneNumber,
+  parsePhoneNumberFromString,
+} from "libphonenumber-js";
+import { useFormContext, Controller, RegisterOptions } from "react-hook-form";
+import snakeCase from "lodash/snakeCase";
+import { useTranslation } from "react-i18next";
+
+import { FormField } from "../form_field";
+import { useFormField } from "../use_form_field";
+import PhoneInput, { PhoneInputProps } from "./phone_input";
+
+interface OwnProps {
+  defaultValue?: string;
+  label?: Maybe<string>;
+  name: string;
+  rules?: Exclude<RegisterOptions, "valueAsNumber" | "valueAsDate">;
+}
+
+export type PhoneFieldProps = OwnProps &
+  Omit<PhoneInputProps, "onChange" | "defaultValue">;
+
+const formattedValue = (value: string): string => {
+  const parsedPhoneNumber = parsePhoneNumberFromString(value);
+  return parsedPhoneNumber?.formatNational() ?? "";
+};
+
+export const PhoneField: React.FC<PhoneFieldProps> = ({
+  defaultValue = "",
+  label,
+  name,
+  rules,
+  ...inputProps
+}): React.ReactElement => {
+  const { control } = useFormContext();
+  const { error, label: formFieldLabel } = useFormField({
+    name,
+    buildLabel: label === undefined,
+  });
+  const { t } = useTranslation("errors");
+  const fieldLabel = label ?? formFieldLabel;
+  const inputId = snakeCase(name);
+  const placeholder = "";
+
+  const phoneNumberValidation = (phoneNumber: string): string | undefined => {
+    const valid = isPossiblePhoneNumber(phoneNumber, "US");
+
+    if (!valid) return t("phoneNumber") || undefined;
+  };
```

Comment:
I am unsure how this bit should be ported over to work correctly. It does not throw any errors in Storybook, so it seems like it may be off. 

I tinkered a bit with the `form_error` component, but wonder if it could be an issue leveraging `FormField` and `Controller` together.